File: Leet Code/JavaScript/problem1281.js
const subtractProductSum = (num) => {
    let sum = 0;
    let product = 1;

    while (num > 0) {
        sum += (num % 10);
        product *= (num % 10);
        num = Math.floor(num / 10);
    };
    return product - sum;
};

console.log("Result is ", subtractProductSum(4421));

File: Leet Code/Python/problem1281.py
def subtractProductSum(num):
    sum = 0
    product = 1

    while (num > 0):
        sum += (num % 10)
        product *= (num % 10)
        num = num // 10
    
    return product - sum

myNum = int(input("Enter an Integer: "))
result = subtractProductSum(myNum)
print("Result is ", result)